src/PrestaShopBundle/Form/Admin/Improve/International/Tax/TaxOptionsType.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShopBundle\Form\Admin\Improve\International\Tax;
+
+use PrestaShop\PrestaShop\Core\Form\FormChoiceProviderInterface;
+use PrestaShopBundle\Form\Admin\Type\SwitchType;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+/**
+ * Handles "Improve > International > Taxes" options form
+ */
+class TaxOptionsType extends AbstractType
+{
+    /**
+     * @var array
+     */
+    private $taxOptionsConfiguration;
+
+    /**
+     * @var FormChoiceProviderInterface
+     */
+    private $taxAddressTypeChoiceProvider;
+
+    /**
+     * @var FormChoiceProviderInterface
+     */
+    private $taxRuleGroupChoiceProvider;
+
+    public function __construct(
```

Comment:
missing docblock

Reply:
more than the Doc block constructor: I'd like to see the properties described, at least the first one as it's not an object and so one I can't figure out by myself which data I can expect from it

Reply:
For this (and other cases where form field names are reused) I had a thought maybe its worth to have field names defined as constants in related formType?
There are already couple places during form building,handling, etc. where these values are reused.
wdyt?